### ISSUE TITLE
a11y: Missing textual descriptions for interactive image elements

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12805]
a11y Issue: 83

The textual description is not specified for the interactive image that resulted in screen reader users finding it difficulty to understand the purpose of the interactive image.

The following places need to be fixed:

Account and API Keys
 - API Keys section
  - Copy/ Copied interactive images 

Kubectl Shell
- Close interactive images associated with Kubectl: local text
- Resize interactive image
- Copy/ Copied interactive images 

Projects/Namespaces
- Favorite 

**Example Usage that needs fixing:**

![image](https://github.com/user-attachments/assets/faa26245-b221-4861-8575-d0393dbb9787)

**Remediation:**

Specify accurate and descriptive textual description such as "Copy Link"/"Link Copied" for the interactive image using hidden text within <a> element. Ensure that the textual description gets updated on user interaction via scripting.

**Comment by izaac** [comment]
Chrome version: `133.0.6943.127`
Firefox version: `135.0.1`
Rancher version: `v2.11-3fd614ce7cab69d448bf39570fbe8b657112d710-head`
UI version: `master 470c849`

Using a screen reader extension I validated the text is mentioned when selecting the items described in the issue. ✅
